doc/guides/device_mgmt/index.rst

```diff
@@ -39,14 +39,466 @@ functionality over BLE and serial.
+    must be set.
+
+.. _image_mgmt:
+
+Image Management
+****************
+
+The image management provided by ``mcumgr`` is  based on the image format defined
+by MCUboot. For more details on the internals see `MCUboot design`_ and :ref:`west-sign`.
+
+To list available images in a device::
+
+  mcumgr <connection-options> image list
+
+This should result in an output similar to this::
+
+  $ mcumgr -c acm0 image list
+  Images:
+    image=0 slot=0
+      version: 1.0.0
+      bootable: true
+      flags: active confirmed
+      hash: 86dca73a3439112b310b5e033d811ec2df728d2264265f2046fced5a9ed00cc7
+  Split status: N/A (0)
+
+Where ``image`` is the number of the image pair in a multi-image system, and slot
+is the number of the slot where the image is stored, ``0`` for primary and ``1`` for
+secondary. This image being ``active`` and ``confirmed`` means it will run again on
+next reset. Also relevant is the ``hash``, which is used by other commands to
+refer to this specific image when performing operations.
+
+An image can be manually erased using::
+
+  mcumgr <connection-options> image erase
+
+The behavior of ``erase`` is defined by the server (``mcumgr`` in the device).
+The current implementation is limited to erasing the image in the secondary
+partition.
+
+To upload a new image::
+
+  mcumgr <connection-options> image upload [-n] [-e] [-u] <signed-bin>
+
+* ``-n``: This option allows uploading a new image to a specific set of images
+  in a multi-image system, and is currently only supported by MCUboot when the
+  ``BOOT_SERIAL`` option is enabled.
+
+* ``-e``: This option avoids performing a full erase of the partition before
+  starting a new upload.
+
+.. tip::
+
+   The ``-e`` option should always be passed in because the ``upload`` command
+   already checks if an erase is required, respecting the
+   :option:`CONFIG_IMG_ERASE_PROGRESSIVELY` setting.
+
+.. warning::
+
+   ``mcumgr`` does not understand .hex files, when uploading a new image always
+   use the .bin file.
+
+* ``-u``: upgrade only to newer image version.
+
+After an image upload is finished, a new ``image list`` would now have an output
+like this::
+
+  $ mcumgr -c acm0 image upload -e build/zephyr/zephyr.signed.bin
+    35.69 KiB / 92.92 KiB [==========>---------------]  38.41% 2.97 KiB/s 00m19
+
+Now listing the images again::
+
+  $ mcumgr -c acm0 image list
+  Images:
+   image=0 slot=0
+    version: 1.0.0
+    bootable: true
+    flags: active confirmed
+    hash: 86dca73a3439112b310b5e033d811ec2df728d2264265f2046fced5a9ed00cc7
+   image=0 slot=1
+    version: 1.1.0
+    bootable: true
+    flags:
+    hash: e8cf0dcef3ec8addee07e8c4d5dc89e64ba3fae46a2c5267fc4efbea4ca0e9f4
+  Split status: N/A (0)
+
+To test a new upgrade image the ``test`` command is used::
+
+  mcumgr <connection-options> image test <hash>
+
+This command should trigger a ``test`` upgrade, which means that after the next
```

Comment:
trigger -> request, mark. I think `to trigger` mean that `an action` will happen immediately after that, which is incorrect in this case.

Reply:
Done.